### PR TITLE
tests added for pcsa view

### LINKED ISSUE #58: Write tests for PCSA
To test proper functioning of the app.

### PR DESCRIPTION
**Pull request by muddlebee**
fixes #58

### CODE REVIEW

File: pcsa/tests.py [hunk 1]
```diff
@@ -1,3 +1,143 @@
...
+
+from pcsa.models import PcsaPost
+from signup.models import Pcuser
 
 # Create your tests here.
+
+class PcsaTests(TestCase):
+
+    def setUp(self):
+        self.u1 = User.objects.create_superuser(username='admin',
```
Comment (medhach) [inline pcsa/tests.py]: Can you make it user1? Its more readable that way. :)

File: pcsa/tests.py [hunk 2]
```diff
@@ -1,3 +1,143 @@
...
+
+        self.u2 = User.objects.create_superuser(username='admin2',
+                                                password='password2',
+                                                email='')
+
+        self.u1.save()
+        self.u2.save()
+
+        self.o1 = Pcuser(user=self.u1)
+        self.o2 = Pcuser(user=self.u2)
```
Comment (medhach) [inline pcsa/tests.py]: Same.

**Comment by muddlebee** [comment]
@medhach please review

**Comment by medhach** [comment]
@anweshknayak Good job! Could you squash your commits into one? That way its less clumsy to merge to the master branch.

**Comment by muddlebee** [comment]
@medhach I am doing it. I misunderstood. I was actually smashing all PRs into one. My bad

**Pull request closed without merging**